src/EventReporter/ConsoleSummaryEventReporter.php

```diff
@@ -106,7 +100,7 @@ private function printSummary(array $events, OutputInterface $output): void
     /**
      * @psalm-param list<EventInterface> $events
      * @psalm-param list<class-string> $desiredEvents
-     * @var EventInterface[] $events
+     * @param EventInterface[] $events
```

Comment:
it seems it can in separate PR ?

Reply:
Yes, it could. I fixed it in here, since test were failing. Would you like to continue or should I provide this change in a separate PR?

Reply:
I cherry-picked your commit at PR https://github.com/laminas/laminas-servicemanager-inspector/pull/6

Reply:
@arueckauer since the commit is at PR #6 now, could you remove the commit? Thank you.